tls/s2n_key_update.c

```diff
@@ -146,3 +146,12 @@ int s2n_check_record_limit(struct s2n_connection *conn, struct s2n_blob *sequenc
 
     return S2N_SUCCESS;
 }
+
+int s2n_connection_request_key_update(struct s2n_connection *conn, bool peer_update_requested)
+{
+    POSIX_ENSURE_REF(conn);
+    /* s2n-tls does not currently support requesting key updates from peers */
+    POSIX_ENSURE(!peer_update_requested, S2N_ERR_UNIMPLEMENTED);
```

Comment:
I looked it up to double-check. S2N_ERR_UNIMPLEMENTED is S2N_ERR_T_INTERNAL. That doesn't seem right here. You should probably pick a usage error.